src/modules/Security.rb

```diff
@@ -365,6 +340,73 @@ def read_kernel_settings
       end
     end
 
+    def read_encryption_method
```

Comment:
NP: Ideally these methods should be private and only the main `Read` should be public. But there already are other public read methods so let's keep it as it is...